packages/core/src/services/zoom.ts

```diff
@@ -64,4 +64,22 @@ export class Zoom extends Service {
 			extent(zoomBarData, (d: any) => d[domainIdentifier])
 		);
 	}
+
+	isZoomBarLoading(position) {
+		return Tools.getProperty(
+			this.model.getOptions(),
+			"zoomBar",
+			position,
+			"loading"
+		);
+	}
+
+	isZoomBarLocked(position) {
+		return Tools.getProperty(
+			this.model.getOptions(),
+			"zoomBar",
+			position,
+			"locked"
```

Comment:
wouldn't just `options.zoomBar.position.loading === false` mean that it should be locked? do we really need the extra flag?

Reply:
`loading` means we should display the skeleton of zoom bar.
`locked` means we should display zoom bar as usually, but only read only, no zoom bar handle manipulation allowed.